frontend/src/components/layouts/MainLayout.tsx

```diff
@@ -1,12 +1,14 @@
 import { Layout } from 'antd';
 
-const { Sider, Header, Content } = Layout;
+import { Header } from '.'
```

Comment:
I wtedy dzięki temu tutaj zrobisz takiego importa: 

`import { Header } from './components'`

Od razu jest wiadome, że Header to wewnętrzny podkomponent dla szablonów.

Dodatkowo importując z `.` narażasz się na `Circular Dependency`, które może powodować trochę problemów.